STM32/Util/Src/systeminfo.c

```diff
@@ -111,6 +120,43 @@ const char* systemInfo()
     return buf;
 }
 
+const char* statusInfo()
+{
+    static char buf[600] = { 0 };
```

Comment:
Optimisation: Could save 600 bytes memory space by combining this with the buffer from systemInfo